src/expr/skolem_manager.h

```diff
@@ -228,9 +228,19 @@ std::ostream& operator<<(std::ostream& out, SkolemFunId id);
 
 /**
  * A manager for skolems that can be used in proofs. This is designed to be
- * a trusted interface to NodeManager::mkSkolem, where one
- * must provide a definition for the skolem they create in terms of a
- * predicate that the introduced variable is intended to witness.
+ * a trusted interface for constructing variables of SKOLEM type, where one
+ * must provide information that characterizes the skolem. This information
+ * may either be:
+ * (1) the term that the skolem purifies (mkPurifySkolem)
```

Comment:
```suggestion
 * (1) the term that the skolem purifies (mkPurifySkolem())
```

Reply:
I'd prefer not having these parentheses, since the method takes arguments, the text is referring to the method name.

Reply:
I know, but it is easier to parse if it is immediately clear that this is a function name (hence my suggestion to add the parens). Note that this is common syntax, e.g., for Doxygen, it is enough to use empty parenthesis (except if you want to refer to a specific overload) for it to convert this to a link to the function declaration documentation. It's not a hill I'm willing to die on, though.

Reply:
Hm, this surprises me.  I'd prefer a different convention like using quotes, e.g. `mkPurifySkolem`. I think we do this in the API?

Reply:
If we do it's wrong, because then Doxygen can't link to it.